src/movieList.js

```diff
@@ -0,0 +1,48 @@
+import likeImage from './assets/Like-emoji.png';
+
+const createList = async () => {
+  const container = document.querySelector('.home');
+
+  const response = await fetch('https://api.tvmaze.com/shows');
+  const data = await response.json();
+
+  const filtered = data.filter((display) => display.id <= 12);
+
+  filtered.forEach((content) => {
+    const movieCard = document.createElement('div');
+    movieCard.className = 'card';
+
+    const picture = document.createElement('img');
+    picture.src = content.image.medium;
```

Comment:
I would suggest you give the images an 'alt' value. This is to assist people who might be using technologies like screen readers.